namerd/core/src/main/scala/io/buoyant/namerd/DtabStore.scala

```diff
@@ -54,6 +54,8 @@ object DtabStore {
 
   class DtabNamespaceInvalidException(ns: Ns)
     extends Exception(s"invalid dtab namespace $ns: namespace must contain only letter, number or '-' characters")
+  class DtabContainsInvalidDentriesException(dentries: IndexedSeq[Dentry])
```

Comment:
IndexedSeq[Dentry] renders weirdly in the exception message: `Vector(null, Dentry(/foo=>/#/io.l5d.fs))`

What about having this just be the input string that triggered the exception?  e.g. req.contentString